### PR TITLE
Added small refactorings, mostly code styling

### PR DESCRIPTION
**Pull request by larssmit**
Suppress Ruby warnings + use add_dependency in gemspec + removed require rubygems

Fixed merge leftovers

### CODE REVIEW

File: lib/zewo.rb [hunk 1]
```diff
@@ -104,20 +94,16 @@ def build_dependencies
         dependency_repos = File.read(dir('Package.swift')).scan(/(?<=Zewo\/|SwiftX\/|VeniceX\/|paulofaria\/)(.*?)(?=\.git)/).map(&:first)
 
         # here we should do something like dependency_repos -= ['CMySQL', 'COpenSSL', 'CLibpq']
-
+        dependency_repos = File.read(dir('Package.swift')).scan(%r{/(?<=Zewo\/)(.*?)(?=\.git)/}).map(&:first)
```
Comment (paulofaria) [inline lib/zewo.rb:97]: I think this line shouldn't be here. The comment above can be deleted as well. It was solved somewhere else.

File: lib/zewo/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module Zewo
-  VERSION = "0.2.9"
+  VERSION = "0.2.9".freeze
```
Comment (paulofaria) [inline lib/zewo/version.rb:2]: out of curiosity what does this `freeze` do?

File: lib/zewo.rb (hunk 1 above)
Comment (Danappelxx) [inline lib/zewo.rb:97]: That's right - this line should be deleted.

File: lib/zewo/version.rb (hunk 2 above)
Comment (Danappelxx) [inline lib/zewo/version.rb:2]: I'm assuming it makes the string immutable (sort of like `let` in swift).

Comment (larssmit) [inline lib/zewo/version.rb:2]: Indeed, it makes an object immutable like the VERSION constant (everything in Ruby is an object). For example:

~ $ irb
irb(main):001:0> a = "test"
=> "test"
irb(main):002:0> a << "change"
=> "testchange"
irb(main):003:0> a.freeze
=> "testchange"
irb(main):004:0> a << "change"
RuntimeError: can't modify frozen String
    from (irb):4
    from /Users/larssmit/.rbenv/versions/2.3.0/bin/irb:11:in `<main>'
irb(main):005:0>

File: lib/zewo.rb (hunk 1 above)
Comment (larssmit) [inline lib/zewo.rb:97]: I will remove the line.

**Pull request closed without merging**